src/python/test/test_dxclient.py

```diff
@@ -5831,6 +5831,19 @@ def test_remote_build_app(self):
         app_dir = self.write_app_directory("minimal_remote_build_åpp", json.dumps(app_spec), "code.py")
         run("dx build --remote --app " + app_dir)
 
+    def test_remote_build_app_trusty(self):
+        app_spec = {
+            "name": "minimal_remote_build_app_trusty",
+            "dxapi": "1.0.0",
+            # Use a package specific to trusty but not in precise as part of the execdepends to ensure it is installed properly
+            "runSpec": {"file": "code.py", "interpreter": "python2.7", "distribution": "Ubuntu", "release": "14.04", "execDepends": [{"name": "postgresql-9.3"}], "systemRequirements": {"*": {"instanceType": "mem1_ssd1_x4"}}},
```

Comment:
Test needs to use "buildDepends" since that's what the builder app is going to look at. (I think it ignores "execDepends", so I suspect this test would pass as written even if the build happened on 12.04.)